Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorConnection.cpp

```diff
@@ -6,14 +6,15 @@
  *
  */
 
+#include "AzFramework/Spawnable/InMemorySpawnableAssetContainer.h"
```

Comment:
Standard: incorrect inclusion style.
It might also be an idea to sort the includes by alphabet to make it easier to spot duplicate inclusions.